testsuite/features/profiles/Kiwi/POS_Image-JeOS6_41/config.sh

```diff
@@ -1,142 +0,0 @@
-#!/bin/bash
```

Comment:
I think we need to also delete the source in the other repository.
Otherwise, next time we generate again these profiles, it will come to our plate again.

Reply:
This is the source, isn't it ?

Reply:
I'm not exactly sure where is the Makefile, but I know we have one, and the source is not in our Spacewalk or Uyuni repositories, but in a separate repository.

Reply:
Here is the Makefile, but it is internal:

https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/profiles/Makefile

It populates the NUE and µPRV directories with a slight edition of the files.

Reply:
All clear! Thanks!